website/blog/2025-03-19-20-must-try-mongodb-queries-operations-ferretdb.md

```diff
@@ -0,0 +1,436 @@
+---
+slug: 20-must-try-mongodb-queries-operations-ferretdb
+title: '20 must-try MongoDB operations and queries on FerretDB'
+authors: [alex]
+description: >
+  FerretDB just got better with the release of v2, bringing deeper MongoDB compatibility, and enabling more advanced workloads to run complex queries for most use cases. Find out in this post.
+image: /img/blog/mongodb-operations-ferretdb.jpg
+tags: [observability, product, open source]
+---
+
+![20 Must-Try Advanced MongoDB Queries on FerretDB](/img/blog/mongodb-operations-ferretdb.jpg)
+
+FerretDB just got better with [the release of v2](https://blog.ferretdb.io/ferretdb-releases-v2-faster-more-compatible-mongodb-alternative/), bringing deeper MongoDB compatibility, and enabling more advanced workloads to run complex queries for most use cases.
+
+<!--truncate-->
+
+Beyond just basic CRUD operations, most developers and businesses work with large datasets with real-time analytical queries, text and vector indexing, geospatial searches, replication, among other features.
+All of these operations are available on FerretDB – in open source and without any danger of vendor lock-in or proprietary limitations.
+
+In this post, we're diving into 20 advanced MongoDB features and operations you can run on FerretDB.
+We will cover basic queries, authentication, indexes, aggregation operations, geospatial queries, time series collections, and more.
```

Comment:
"advanced features" such as "basic queries"? and what about "time series collections"?